plugins/check_curl.c

```diff
@@ -1023,23 +1055,60 @@ check_http (void)
 
   if (strlen (header_expect)) {
     if (!strstr (header_buf.buf, header_expect)) {
+
```

Comment:
i would prefer to not start blocks with a blank line.

Reply:
Indeed, sorry about that.